File: test/functional/JobPortalAutomationTest/src/AddVacancies.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class AddVacancies {

    public static WebDriver add(WebDriver driver) throws InterruptedException {


        WebElement jobName;
        WebElement skillrequired;
        WebElement minRate;
        WebElement minSalary;
        WebElement maxRate;
        WebElement maxSalary;
        WebElement numOfEmployee;
        WebElement jobType;
        WebElement description;
        WebElement addButton;


        try {

            jobName = driver.findElement(By.xpath("//*[@id='JobName']"));
            skillrequired = driver.findElement(By.xpath("//*[@id='SkillReq']"));
            minRate = driver.findElement(By.xpath("//*[@id='MinRate']"));
            minSalary = driver.findElement(By.xpath("//*[@id='MinSalary']"));
            maxRate = driver.findElement(By.xpath("//*[@id='MaxRate']"));
            maxSalary = driver.findElement(By.xpath("//*[@id='MaxSalary']"));
            numOfEmployee = driver.findElement(By.xpath("//*[@id='NoEmp']"));
            jobType = driver.findElement(By.xpath("//*[@id='Job_type_entered']"));
            description = driver.findElement(By.xpath("//*[@id='description']"));
            addButton = driver.findElement(By.xpath("//*[@id='addVacancies']"));

        }
        catch (Exception e){
            throw e;
        }

        jobName.sendKeys("Web Developer");
        skillrequired.sendKeys("Programer");
        minRate.sendKeys("100");
        minSalary.sendKeys("3000");
        maxRate.sendKeys("300");
        maxSalary.sendKeys("9000");
        numOfEmployee.sendKeys("5");
        description.sendKeys("I need new web programmer");
        addButton.click();
        Thread.sleep(2000);


        return driver;

    }
}

File: application/controllers/employer/addVacancies.php
<?php
/**
 * Created by PhpStorm.
 * User: eandualem
 * Date: 6/20/18
 * Time: 3:36 AM
 */

class addVacancies extends Controller
{
    function __construct()
    {
        parent::__construct();
    }

    public function add()
    {
        $JobName = $_POST["JobName"];
        $SkillReq = $_POST["SkillReq"];
        $MinRate = $_POST["MinRate"];
        $MinSalary = $_POST["MinSalary"];
        $Job_type = $_POST["Job_type_entered"];
        $NoEmp = $_POST["NoEmp"];
        $MaxRate = $_POST["MaxRate"];
        $MaxSalary = $_POST["MaxSalary"];
        $Description = $_POST["Description"];


        $temp=array('JobName'=>$JobName,'JobType'=>$Job_type,'Jobdescription'=>$Description, 'SkillRequired'=>$SkillReq, 'maxsalary'=>$MaxSalary, 'minsalary'=>$MinSalary,
            'minrate'=>$MinRate, 'maxrate'=>$MaxRate, 'nemberofemployee'=>$NoEmp);

        $login_model = $this->loadModel('Employer');
        $inserted = $login_model->create_job(json_encode($temp));


        if($inserted)
        {
            try {
                echo "Job Added Successfully";
            } catch (Exception $e) {
                throw new Exception("Error rendering result");
            }
        } else {
            echo "Failed try again";
        }

    }

}